lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -130,17 +139,17 @@ MayaUsd::LayerManager* findNode()
     for (; !iter.isDone(); iter.next()) {
         MObject mobj = iter.item();
         fn.setObject(mobj);
-        if (fn.typeId() == MayaUsd::LayerManager::typeId && !fn.isFromReferencedFile()) {
+        if (fn.typeId() == MayaUsd::LayerManager::typeId && isFromReference(fromReference, fn)) {
             layerManagerHandle = mobj;
             return static_cast<MayaUsd::LayerManager*>(fn.userNode());
         }
     }
     return nullptr;
 }
 
-MayaUsd::LayerManager* findOrCreateNode()
+MayaUsd::LayerManager* findOrCreateNode(const MFnReference* fromReference)
```

Comment:
Most of the changes are related to adding this new fromReference parameter to a few functions.